builder/builder_configuration.yaml

```diff
@@ -15,10 +15,12 @@ processors:
   - import: go.opentelemetry.io/collector/processor/batchprocessor
     gomod: go.opentelemetry.io/collector v0.60.0
   - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.60.0"
-  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor v0.60.0"
```

Comment:
Why replace? We should keep both there IMO, and add only the one we care about in the build process.

Reply:
why add a processor we don't use?